kafka/src/main/java/com/metamx/tranquility/kafka/model/TranquilityKafkaConfig.java

```diff
@@ -0,0 +1,58 @@
+package com.metamx.tranquility.kafka.model;
+
+import com.google.common.collect.Lists;
+import com.metamx.tranquility.config.TranquilityConfig;
+import org.skife.config.Config;
+import org.skife.config.Default;
+import scala.collection.JavaConversions;
+
+/**
+ * Configuration object which extends Tranquility configuration with Kafka specific parameters.
+ */
+public abstract class TranquilityKafkaConfig extends TranquilityConfig
+{
+  public TranquilityKafkaConfig()
+  {
+    GlobalProperties_$eq(
```

Comment:
_$eq is the internal setter created by Scala. Giving TranquilityConfig a constructor sounds like a good option.